PullRequestCreated/ReadMe.md

```diff
@@ -7,6 +7,10 @@ This action will take the first non-`Sub-task` ticket and use it as a parent for
 - if the parent is not an `Epic`, the created ticket will be a `Sub-task`
 - if there is no parent, the created ticket will be a `Task`
 
+This action will also update the PR title and description to mention the right Jira tickets.
```

Comment:
I don't understand what this means. It looks like a combination of the next two points.
```suggestion
```